CMakeLists.txt

```diff
@@ -41,8 +41,15 @@ include(${TORCH_XPU_OPS_ROOT}/cmake/BuildFlags.cmake)
 
 option(USE_XCCL "Build with XCCL support" OFF)
 
-if(NOT WIN32 AND USE_XCCL)
+if(USE_XCCL)
   include(${TORCH_XPU_OPS_ROOT}/cmake/XCCL.cmake)
+  if(NOT PYTORCH_FOUND_XCCL)
```

Comment:
refer to https://github.com/pytorch/pytorch/blob/d9cf1debf9831edac51aab7ca29d8a0584474cc5/cmake/Dependencies.cmake#L94